File: src/main/org/slug/CSSLoader.java
package org.slug;

import java.io.*;

public class CSSLoader {
    private static String EMPTY = "";
    public String getCSS() {
        try {
            ClassLoader classLoader = getClass().getClassLoader();
            InputStream in = classLoader.getResourceAsStream("style.css");
            StringBuffer sb = new StringBuffer();
            BufferedReader br = new BufferedReader(new InputStreamReader(in,"UTF-8"));
            for (int c = br.read(); c != -1; c = br.read()) {
                sb.append((char) c);
            }
            return sb.toString();
        } catch (UnsupportedEncodingException e) {
            return EMPTY;
        } catch (IOException e) {
            return EMPTY;
        }

    }
}

File: src/main/kotlin/org/slug/CSSLoader.kt
package org.slug

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader
import java.io.UnsupportedEncodingException

class CSSLoader {
    fun loadCSS(): String {
        val EMPTY = ""
        try {
            val classLoader = javaClass.classLoader
            val inputReader = classLoader.getResourceAsStream("style.css")
            val buffer = StringBuffer()
            val bufferedReader = BufferedReader(InputStreamReader(inputReader, "UTF-8"))
            var character = bufferedReader.read()
            while (character != -1) {
                buffer.append(character.toChar())
                character = bufferedReader.read()
            }
            return buffer.toString()
        } catch (e: UnsupportedEncodingException) {
            return EMPTY
        } catch (e: IOException) {
            return EMPTY
        }
    }
}
